packages/profiling-node/src/cpu_profiler.ts

```diff
@@ -54,6 +54,13 @@ export function importCppBindingsModule(): PrivateV8CpuProfilerBindings {
     return createdRequire(`${binaryPath}.node`);
   }
 
+  // @electron/rebuild generates the binary in the generic node-gyp directory
+  try {
+    return createdRequire('../../build/Release/sentry_cpu_profiler.node');
```

Comment:
Does it actually place it here? I though it moves it into pkg/bin/$platform-$abi/smth.node?

Reply:
It actually outputs binaries in both locations!

On macOS I get a binaries at:
- `./bin/darwin-arm64-132/profiling-node.node`
- `./build/Release/sentry_cpu_profiler.node`

The binaries are identical and work in Electron and the `build/Release` path doesn't vary by platform/arch so I chose that one 🤷‍♂️

The `@electron/rebuild` tests at least point to similar paths:
https://github.com/electron/rebuild/blob/8e8e6f94f8bac41044cf30b7898bdcaa04df0a2f/test/rebuild.ts#L150

Reply:
Ahh, that is good to know! I dont have a clean cmd so I usually always have some version of the binary that is built via postinstall or something in the build/release repo.

Reply:
I think this condition that you added here will mean that we always prefer this freshly built binary, which may have been built outside of just electron/rebuild, like for example someone running node-gyp directly. In any case, I think this is fine.

Reply:
Ah yes. Maybe we do a check for Electron first?

Checking for `process.versions.electron` is probably enough...

Reply:
I think we should, yes. 

We should check to ensure that we default ship this build/release binding or that one is always generated as part of postintall, as it will otherwise break setups that bundle .node files as the path is statically analyzable, but the file wont actually exist which will result in an error

Reply:
> otherwise break setups that bundle .node files as the path is statically analyzable, but the file wont actually exist

Mmmm... not sure what to do about that. Can we rely on the `install` script running? I guess not always!

Maybe we could already have a zero byte file at `./build/Release/sentry_cpu_profiler.node` so that bundlers don't complain about it missing but it gets overwritten. 

Then if you're running in Electron and `@electron/rebuild` hasn't run, the loading with throw and we catch it and explain in the debug message that you need to run `@electron/rebuild`?